src/containers/Member.js

```diff
@@ -134,6 +136,7 @@ const MemberPage = ({
         window.location.href = getSignInHref(currentHref)
       }, 2000)
     }
+    getUserData(jwt, userID)
```

Comment:
could we add a blank before this line or add a comment to show its purpose ?
IMO, it looks like part of the check authorization section now 😂

Reply:
fixed